skimage/color/colorconv.py

```diff
@@ -808,7 +811,7 @@ def xyz2lab(xyz, illuminant="D65", observer="2"):
     return np.concatenate([x[..., np.newaxis] for x in [L, a, b]], axis=-1)
 
 
-def lab2xyz(lab, illuminant="D65", observer="2"):
+def lab2xyz(lab, illuminant="D65", observer="2", error_on_invalid=False):
```

Comment:
Isn't there already a way to do this with warnings?  Such as the catch_all filter?